core/frontend/src/ViewState.ts

```diff
@@ -161,7 +161,7 @@ export abstract class ViewState extends ElementState {
         return;
 
     if (this.isAttachedToViewport)
-      this.onDisplayStyleChanged.raiseEvent();
+      this.onDisplayStyleChanged.raiseEvent(style);

```

Comment:
Is there any way to template-ize a variable list of arguments? There's a (poorly named) class called BeUiEvent that takes a listener with a single strongly-typed argument. Instead of `raiseEvent` you call `emit`.

Reply:
A while ago I prototyped a type-safe BeEvent. I don't appear to have saved the code.

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype